Resources/Prototypes/DeltaV/tags.yml

```diff
@@ -41,3 +41,6 @@
 
 - type: Tag
   id: Skirt
+  
+- type: Tag
+  id: Gavel
```

Comment:
Tags should be alphabetical + line at the end

Reply:
Fixed up